subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/ivyresolve/verification/writer/WriteDependencyVerificationFile.java

```diff
@@ -212,10 +215,12 @@ private void serializeResult(SignatureVerificationService signatureVerificationS
             verificationsBuilder.setVerifySignatures(true);
         }
         DependencyVerifier verifier = verificationsBuilder.build();
-        DependencyVerificationsXmlWriter.serialize(
-            verifier,
-            new FileOutputStream(out)
-        );
+        if (isWriteVerificationFile()) {
```

Comment:
Unrelated to that, but I guess in `writeEntriesSerially();` we never print verification errors in case we just export keys right, since checksums are empty and nothing is added to `entriesToBeWritten`?

If that is the case, then good

Reply:
I think that's right. I'll add a test!